docs/development/reviewing-patches.rst

```diff
@@ -33,6 +33,32 @@ These are small things that are not caught by the automated style checkers.
 * Does a variable need a better name?
 * Should this be a keyword argument?
 
+Testing with twine
+------------------
+If you are testing Warehouse locally, you may want to use `twine`_ to try
+uploading a test package. First, checkout the branch you would like to test.
+Then, start up the development environment (as described in :ref:`getting-started`).
+Once you have the Warehouse site working on ``localhost:80``, you can upload a
+package to the version of Warehouse running in your development environment
+with Twine using the following command:
+
+.. code-block:: console
+
+  twine upload --repository-url http://localhost/legacy/ -u username -p password path to distribution
```

Comment:
Can you change this to:
```
twine upload --repository-url http://localhost/legacy/ -u <username> -p password <path to distribution>
```
The angle brackets indicate that the value needs to be replaced by the user.